9.2.15 Release

su-sws-jira opened the pull request:
# DO NOT DELETE

su-sws-jira committed:
9.2.15

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr1000-rnqygp3lk7rmomu8r42sq583fqnozrey.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/618d6d71e36865b8cb3b61e7

Commit:
Hotfix(STN-924): going back to using animationFrame api instead of eventListeners as (#1003)

* fix: going back to using animationFrame api instead of eventListeners as
it seems to have been disabling the window's ability to scroll.

- add back scroll function

* chore: compile JS

pookmish committed:
Publications importer skip on empty processes

joegl committed:
Added physics, art, humanrights, and anthro_net to multi-site. (#1005)

Co-authored-by: joegl <joegl@users.noreply.github.com>

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr1000-rnqygp3lk7rmomu8r42sq583fqnozrey.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/618d6d71e36865b8cb3b61e7
